src/ms.tex

```diff
@@ -303,13 +303,54 @@ \section{Learn Astropy} \label{sec:learn}
 
 \secauthor{Lía Corrales, David Shupe + Learn team}
 
+\subsubsection{Current status and scope}
+
+The Astropy-ii paper discussed the {\it Learn Astropy} effort only in the
+context of online tutorials and guides. This vision has been realized as
+{\it Learn Astropy} has released a new version of this website, and has
+expanded its efforts
+in user engagement and in conducting workshops. The Learn team meets
+weekly to drive these issues forward.
+
 \begin{itemize}
-\item Current status and scope
-\item Vision for the future (review what we said in v2.0 paper)
-\item User forums and engagement with user base
-\item Workshops
+
+\item {\bf Learn Astropy website:} The site has a new search interface powered by
```

Comment:
Jonathan Sick will add a paragraph here.

Reply:
@jonathansick here's the spot needing your inputs

Reply:
suggested text added.